wavin_ahc9000.py

```diff
@@ -375,6 +375,24 @@ def party_temperature(self, temperature: float):
     def desired_temperature(self) -> float:
         return self.__read_temp(16)
 
+class _ChannelCategory:
+    def __init__(self, channel: int, modbus: _WavinAHC9000Modbus):
+        self.__modbus = modbus
+        self.channel = channel
+        self._category = modbus.Category.channels
+
+    def _read_channel(self):
+        return self.__modbus.read_register(self._category, 0, self.channel, 4)
+
+    @property
+    def output_on(self) -> bool:
+        time = self._read_channel()[0]
+        return time & 16 > 0
```

Comment:
The value returned should already be 16 bit. `return time > 0`

Reply:
As I understand, the `& 16` is bitwise and, and will give the access to 4. bit.
![wavin modbus specification 01072013 - timer event](https://user-images.githubusercontent.com/2368/45028735-790e9c00-b046-11e8-8f33-ae4161f7e041.png)

Reply:
You are right. Don't know what I was thinking. For bit operations I prefer the mask to be hex or binary. `time & 0x10` or `time & 0b10000` or `(time >> 4) & 0x01`.